paper-dialog-behavior.html

```diff
@@ -140,12 +140,6 @@
           this.__prevNoCancelOnEscKey;
         this.withBackdrop = this.withBackdrop && this.__prevWithBackdrop;
       }
-
-      if (this.opened) {
```

Comment:
not needed anymore (forgot to rebase https://github.com/PolymerElements/paper-dialog-behavior/pull/63 before merging it :x)